### ISSUE TITLE
Game breaks after after bar scene leading to first army camp visit

### ISSUE THREAD

**Issue by Brody1018** [JohN100x1/WOTR_IsekaiMod#55]
After downloading your most recent mod version I've come across a bug where I am no longer able to arrive at the army camp because the game breaks and sends me back to the main menu. If it matters, I am using a God Emperor and I have Unlimited Power as my level 1 overpowered ability and I just used my mythic ability choice to get  the Supreme Being overpowered ability. 

**To Reproduce**
Finish wiping the floor with Minagho.
Exit the area through the door.
Read (or don't, your choice) the bar scene revelry.
Try (and fail) to arrive at the army camp.

**Expected behavior**
I expect to be able to arrive at the army camp.

**Screenshots**
![20221214141924_1](https://user-images.githubusercontent.com/120598113/207695705-7000d3fb-b36a-4233-93f6-5caf1c42c740.jpg)

**Additional context**
I was able to get through this part of the game content using version 2.4.0 as late as this past Saturday/Sunday. The only thing that has changed in my mod list is the newest version of the Isekai mod and I did uninstall all of my other mods and ran with only the Isekai mod to make sure it would still happen (it does). I'd check to see if I can get past this point without the Isekai mod installed, but unfortunately I don't have my normal face packs installed on this laptop so I cannot load an earlier save from before I downloaded your mod :(

**Comment by JohN100x1** [comment]
show me your game logs.
you can find it in `\AppData\LocalLow\Owlcat Games\Pathfinder Wrath Of The Righteous`

**Comment by Brody1018** [comment]
[GameLog.txt](https://github.com/JohN100x1/WOTR_IsekaiMod/files/10232416/GameLog.txt)
[GameLogFull.txt](https://github.com/JohN100x1/WOTR_IsekaiMod/files/10232417/GameLogFull.txt)

**Comment by JohN100x1** [comment]
It says you have a missing item in your inventory. My mod doesn't add any items into the game. This doesn't seem like it's related to my mod?
Are you missing items you're supposed to have?

```
[70.4280 - Entity]: Object reference not set to an instance of an object
  at Kingmaker.Items.ItemsCollection.get_IsPlayerInventory () [0x0000b] in <2686087615644cba8590b42c7edeb1a6>:0 

  at Kingmaker.Items.ItemsCollection.DeltaWeight (System.Single deltaWeight) [0x00000] in <2686087615644cba8590b42c7edeb1a6>:0 

  at Kingmaker.Items.ItemsCollection.Extract (Kingmaker.Items.ItemEntity item) [0x00042] in <2686087615644cba8590b42c7edeb1a6>:0 

  at Kingmaker.Items.ItemsCollection.Remove (Kingmaker.Items.ItemEntity item, System.Int32 count) [0x00024] in <2686087615644cba8590b42c7edeb1a6>:0 

  at Kingmaker.Items.ItemsCollection.Remove (Kingmaker.Items.ItemEntity item) [0x00008] in <2686087615644cba8590b42c7edeb1a6>:0 

  at Kingmaker.Items.ItemEntity.OnDispose () [0x0001b] in <2686087615644cba8590b42c7edeb1a6>:0 

  at Kingmaker.EntitySystem.EntityDataBase.Dispose () [0x00016] in <2686087615644cba8590b42c7edeb1a6>:0
```

**Comment by Brody1018** [comment]
The second you said it was related to an item I had a guess of what it might be. When I updated to the most recent version of the mod yesterday I used toybox to reclass my main character. That moved the purple stone knife to my inventory instead of being equipped in a specific spot on my character's loadout. I noticed this earlier so I re-equipped it, but for some reason it didn't register when I was switching zones. All I had to do to fix my issue was to unequip and re-equip the purple stone dagger.

So... I'm at fault, but you updating your good mod is the reason I'm at fault :) Thanks for the support!
